HelloEAR-web/src/main/java/fr/zyra/IndexPageBean.java

```diff
@@ -0,0 +1,49 @@
+package fr.zyra;
+
+import fr.zyra.entities.JeuVideo;
+import fr.zyra.facades.FacadeJeuxVideo;
+import java.io.Serializable;
+
+import javax.faces.view.ViewScoped;
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.FieldDefaults;
+
+@SuppressWarnings("serial")
+@Named
+@ViewScoped
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@NoArgsConstructor
+public class IndexPageBean implements Serializable {
+
+    @Inject
+    FacadeJeuxVideo facadeMetier;
+
+    @Getter
+    @Setter
+    String nomJeu;
+
+    @Getter
+    @Setter
+    String machineJeu;
+
+    @Getter
+    @Setter
+    int anneeSortie;
+
+    public void ajouter() {
+        System.out.printf("%s %s %d", nomJeu, machineJeu, anneeSortie);
+        JeuVideo jv = new JeuVideo();
+        jv.setNom(nomJeu);
+        jv.setMachine(machineJeu);
+        jv.setAnnee(anneeSortie);
+        facadeMetier.save(jv);
+        System.out.println("Sauvegarde effectuée");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)